index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <h2 id="title">DR norman steven</h2>
+      <p>microvere top intern lol</p>
+      <div id="img-div">
+        <img
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt=""
+          id="image"
+        />
+        <div id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </div>
+      </div>
```

Comment:
img-div is better to be a section tag than a div tag. for accessibility and more readability